app/src/main/java/com/dhc/dhcandroid/mvi/EventHandler.kt

```diff
@@ -0,0 +1,3 @@
+package com.dhc.dhcandroid.mvi
+
+typealias EventHandler<T> = (T) -> Unit
```

Comment:
@JeonK1 @fbghgus123 
요렇게 분리해놨는데..괜찮을까요

Reply:
```suggestion
typealias EventHandler<T: UiEvent> = (T) -> Unit
```
괜찮은데 요렇게 하면 더 좋을듯 ??

Reply:
@fbghgus123 typealias 사용할 때 타입 파라미터에 제약 조건을 걸면 안된다네!
기존 코드 그대로 가겠습니다~!

Reply:
오 몰랐넹 이거 출처 공유 가능해?!

Reply:
```suggestion
package com.dhc.dhcandroid.mvi

fun interface EventHandler<T: UiEvent> {
    operator fun invoke (event: T)
}
```
요렇게 대체할 수 있는 방법은 있는듯